### DISCUSSION TITLE
关于 CFW 错误页面 IP 地址的疑问

### DISCUSSION THREAD

**Discussion by ZK6100HGA** [SeaHOH/GotoX#175] [问答/咨询] [answered]
使用CFW代理并遇到CF的出错页面时，出错页面下面显示的IP不是我的，难道使用CFW代理不是直连CF的服务器？

**Comment by SeaHOH** [comment, marked as answer]
是直连，IP 等原始信息被洗掉了，会记录成 Worker fetch 请求的 IP，但仍然存在跟踪 ID。
https://github.com/SeaHOH/GotoX/blob/a41476062e05ebcdd35c0b824b05e4a3ebdb8627/remote/cloudflare_workers/index.js#L212-L215
